src/main/java/org/chtracker/web/PreventiveTreatmentTypeController.java

```diff
@@ -12,25 +10,17 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/TreatmentType")
-public class TreatmentTypeController {
+@RequestMapping("/preventive-treatment-types")
+public class PreventiveTreatmentTypeController {
 
-	private final AbortiveTreatmentTypeRepository abortiveTreatmentTypeRepository;
 	private final PreventiveTreatmentTypeRepository preventiveTreatmentTypeRepository;
 
 	@Autowired
-	public TreatmentTypeController(AbortiveTreatmentTypeRepository abortiveTreatmentTypeRepository, PreventiveTreatmentTypeRepository preventiveTreatmentTypeRepository,
-			PreventiveTreatmentTypeRepository preventiveTreatmentTypeRepository2) {
-		this.abortiveTreatmentTypeRepository = abortiveTreatmentTypeRepository;
-		this.preventiveTreatmentTypeRepository = preventiveTreatmentTypeRepository2;
+	public PreventiveTreatmentTypeController(PreventiveTreatmentTypeRepository preventiveTreatmentTypeRepository) {
+		this.preventiveTreatmentTypeRepository = preventiveTreatmentTypeRepository;
 	}
 
-	@GetMapping("abortive")
-	public Iterable<AbortiveTreatmentType> getAbortiveTreatmentTypes() {
-		return abortiveTreatmentTypeRepository.findAll(Sort.by(new Sort.Order(Direction.ASC, "name")));
-	}
-
-	@GetMapping("preventive")
+	@GetMapping("/")
```

Comment:
In case we add more Http methods maybe it would be good to give them paths based on what they actually are doing.
Perhaps top @RequestMapping over class should be called "/preventatives" and @GetMapping("/get-all-types")

Reply:
the thing is, I don't see what else we can do with preventives/abortive treatments beside showing a least of them.